test/elixir/lib/setup.ex

```diff
@@ -0,0 +1,86 @@
+defmodule Couch.Test.Setup do
+  @moduledoc """
+  Allows to chain setup functions.
+  Example of using:
+
+  ```
+      @tag setup: %Setup{}
+          |> Setup.Start.new([:chttpd])
+          |> Setup.Adapter.new(adapter)
```

Comment:
I think you have removed the adapter code.

Reply:
Good catch. This part of documentation was not updated.

Reply:
fixed